crates/sqlbuiltins/src/builtins.rs

```diff
@@ -34,11 +34,33 @@ pub const POSTGRES_SCHEMA: &str = "pg_catalog";
 /// Schema to store temporary objects (only valid for current session).
 pub const CURRENT_SESSION_SCHEMA: &str = "current_session";
 
-/// First oid available for other builtin objects.
+/// First oid available for other builtin objects that don't have a stable OID.
 ///
-/// All builtin schemas have a stable oid since all objects, including user
-/// objects, rely on the oid of schemas.
-pub const FIRST_NON_SCHEMA_ID: u32 = FIRST_GLAREDB_BUILTIN_ID + 100;
+/// Builtin schemas have stable OIDs since everything (builtin and user objects)
+/// depends on a schema.
+///
+/// Builtin tables have stable OIDs since some depend on data written to disk.
+///
+/// First glaredb builtin OID: 16384
+/// First user object OID: 20000
+///
+/// This means we have ~3600 OIDs to play with for builtin objects. Note that
```

Comment:
why do they have to be sequentially grouped?

Reply:
They don't, it's just easier to track what the next OID should be when adding a table/schema/whatever. If we have more than 16 builtin schemas (unlikely), we can just jump forward to some other available OID.